src/plugins/cosmos/components/CosmosActionButtons/CosmosActionButtons.tsx

```diff
@@ -1,53 +1,68 @@
 import { Flex, FlexProps } from '@chakra-ui/layout'
-import { Button, useColorModeValue } from '@chakra-ui/react'
+import { Button } from '@chakra-ui/react'
 import { Asset } from '@shapeshiftoss/types'
-import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { StakeRoutes } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useHistory } from 'react-router-dom'
 import { Text } from 'components/Text'
-import { useModal } from 'context/ModalProvider/ModalProvider'
 
 type CosmosActionButtonsProps = {
-  activeAction: StakingAction
+  activeRoute: StakeRoutes
   asset: Asset
 }
 export const CosmosActionButtons = ({
-  activeAction,
+  activeRoute,
   asset,
   ...styleProps
 }: CosmosActionButtonsProps & FlexProps) => {
-  const { cosmosStaking } = useModal()
+  const history = useHistory()
 
-  const handleUnstakeClick = () => {
-    cosmosStaking.open({ assetId: asset.caip19, action: StakingAction.Unstake })
+  const handleClick = (route: StakeRoutes) => {
```

Comment:
I prefer to keep things explicit (parental advisory required) with `handleStakeClick`, `handleUnstakeClick` and `handleOverviewClick`